openadapt/config.py

```diff
@@ -132,35 +133,70 @@ def obfuscate(val, pct_reveal=0.1, char="*"):
     return rval
 
 
-
 _OBFUSCATE_KEY_PARTS = ("KEY", "PASSWORD", "TOKEN")
 if multiprocessing.current_process().name == "MainProcess":
     for key, val in dict(locals()).items():
         if not key.startswith("_") and key.isupper():
             parts = key.split("_")
             if (
-                any([part in parts for part in _OBFUSCATE_KEY_PARTS]) and
-                val != _DEFAULTS[key]
+                any([part in parts for part in _OBFUSCATE_KEY_PARTS])
+                and val != _DEFAULTS[key]
             ):
                 val = obfuscate(val)
             logger.info(f"{key}={val}")
 
 
-def filter_log_messages(data):
-    """
-    This function filters log messages by ignoring any message that contains a specific string.
+MAX_NUM_REPEAT_WARNINGS = 3
+MAX_NUM_WARNINGS_PER_SECOND = 5
```

Comment:
chose arbitrary values for now

Reply:
Can you please move to _DEFAULTS so it can be overridden?